python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -213,21 +250,21 @@ def write_images(self, data, butler, njobs=0):
         pathlib.Path(os.path.join(self._output_path, "images")).mkdir(exist_ok=True)
 
         sources = []
+        butler_cache.set(butler, self.config)
```

Comment:
This implicitly assumes that this task is being run single thread. Is that always true?

Reply:
Yes, it's either going to be run in a single thread, or via the multiprocessing starmap. Multithreading here doesn't help because most of the time is spent in matplotlib, not loading data from disk.